### ISSUE TITLE
Output is a labelable element, but accname calculation for output ignores label

### ISSUE THREAD

**Issue by dd8** [w3c/html-aam#435]
Should the accessible name computation for `output` include the `label` element?  
https://w3c.github.io/html-aam/#output-element-accessible-name-computation

It's categorised as labelable in the the HTML Standard:
https://html.spec.whatwg.org/multipage/forms.html#category-label

Chrome/Edge/Opera/Firefox use the label in their accessible name calculation, and use "My label" as the name. Safari doesn't use `label` ~~and uses "This is some output" as the name~~ but maps `output` to AXStaticText which doesn't have a name.

```
<label for="test">My label</label>
<output id="test">This is some output</output>
```

**Comment by scottaohara** [comment]
duplicate of https://github.com/w3c/html-aam/issues/225

pull request is just waiting on people to actually review it https://github.com/w3c/html-aam/pull/402
